src/scripts/tests/comments.test.js

```diff
@@ -0,0 +1,19 @@
+import { commentCounter } from '../modules/comments.js';
+
+describe('Comment section test', () => {
+  test('should return cero if the array is empty', () => {
+    const arr = [];
+    expect(commentCounter(arr.length)).toBe(0);
```

Comment:
[OPTIONAL] It would be better to call the function outside the ```expect``` argument and store it in a temporal variable. That improves readability because it follows the AAA pattern